TodoList/src/app/screens/Checkbox/index.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react'
```

Comment:
`Checkbox` is not a screen, it's a Component.
Ask me if you need more instructions on this.